app/javascript/components/MemberDetailsForm/MemberDetailsForm.js

```diff
@@ -159,15 +154,15 @@ export class MemberDetailsForm extends Component {
           onSubmit={this.submit.bind(this)}
           className="form--big action-form"
         >
-          {this.props.fields.map((field, ii) =>
+          {this.props.fields.map((field, ii) => (
```

Comment:
Seems like `ii` is unused here (although that was also there from the previous version, but worth cleaning up)